network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -6,8 +6,16 @@
 
 {% block page_title %}
     {% environment_prefix %}
-    {% if pageTitle %}{{ pageTitle }}
-    {% else %}{% blocktrans context "“*Privacy Not Included” can be localized. This is a reference to the “*batteries not included” mention on toys." %}*Privacy Not Included | Shop smart and safe{% endblocktrans %} | Mozilla Foundation{% endif %}
+    {# if routable page #}
+    {% if pageTitle %}
+        {{ pageTitle }}
+    {# if custom seo_title for a wagtail page #}
+    {% elif page.seo_title %}
+        {{ page.seo_title }}
+    {% else %}
+    {# default to page.title #}
```

Comment:
For consistency with the comments above, I would swap these two lines. Not a blocker though, just a comment (#nitpick)

Reply:
Good point, I'll swap that out real quick before merge.